website/content/api-docs/system/monitor.mdx

```diff
@@ -13,8 +13,6 @@ some log lines will be dropped.
 
 ## Monitor system logs
 
-@include 'alerts/restricted-admin.mdx'
```

Comment:
I think this was removed erroneously -- this is an endpoint that is enabled by the admin namespace feature, right?